packages/cli/src/ui/components/LoopDetectionConfirmation.test.tsx

```diff
@@ -14,13 +14,15 @@ describe('LoopDetectionConfirmation', () => {
   it('renders correctly', () => {
     const { lastFrame } = renderWithProviders(
       <LoopDetectionConfirmation onComplete={onComplete} />,
+      { width: 101 },
```

Comment:
this test accidentally implicitly had width 101 due to previous tolerant behavior rendering past the terminal width with the test runner and incorrectly dealing with what width 100% + 1 pixel of margin should look like